tmd/Tree/methods.py

```diff
@@ -195,19 +196,19 @@ def get_point_path_distances(self):
     return path_lengths
 
 
-def get_point_trunk_length(self):
+def get_trunk_length(self):
     '''Tree method to get the trunk (first section length).
     '''
     ways, end = self.get_sections_only_points()
     first_section_id = np.where(ways == 0)
     first_section_start = ways[first_section_id]
     first_section_end = end[first_section_id]
-    seg_nums = range(first_section_start[0] + 1, first_section_end[0] + 1)
+    seg_ids = range(first_section_start[0] + 1, first_section_end[0] + 1)
 
-    seg_lengths = get_segment_lengths(self, seg_nums)
+    seg_lengths = get_segment_lengths(self, seg_ids)
     trunk_length = np.sum(seg_lengths)
```

Comment:
Nitpicking but you don't really need to create a variable because it is not used anywhere. Just return the sum directly. Or even 
`return get_segment_lengths(self, seg_ids).sum()`

Reply:
Yeah, I prefer having more steps to make it easier to read and debug. But I will simplify a bit.

Reply:
Sure. It is a matter of taste at this point. For me, this is readable enough to turn it into a one-liner, but it is just a suggestion.